qa/tasks/cephfs/test_nfs.py

```diff
@@ -730,3 +730,12 @@ def exec_cmd_invalid(*cmd):
         exec_cmd_invalid('export', 'info')
         exec_cmd_invalid('export', 'info', 'clusterid')
         exec_cmd_invalid('export', 'apply')
+
+    def test_non_existent_cluster(self):
+        """
+        Test that cluster info doesn't throw junk data for non-existent cluster
+        """
+        cluseter_ls = self._nfs_cmd('cluster', 'ls')
+        self.assertNotIn('foo', cluseter_ls, 'cluster foo exists')
+        cluster_info = self._nfs_cmd('cluster', 'info', 'foo')
+        self.assertIn('cluster does not exist', cluster_info)
```

Comment:
do you know if anything changed for `_nfs_cmd()` recently? anything to the underlying func?

Reply:
this is bound fail since the code returns ENOENT, i was wondering how did it pass in main or squid and the difference is usage of `get_ceph_cmd_stdout()` (in the `_nfs_cmd()` helper) which is used in squid VS `self.mgr_cluster.mon_manager.raw_cluster_cmd(*args)` in quincy. Somehow `get_ceph_cmd_stdout` seems to be suppressing the error

Reply:
> @vshankar this happened because there is a backport of [e03331e](https://github.com/ceph/ceph/commit/e03331e7c4412e33fe55d7a41a22dfeba17cb463) missing. This is backported to squid but not into quincy.

Oh, you mean picked up in squid via branching and not specifically due to backporting, yes?

What about reef branch?

Reply:
reef is good, having the code same as squid https://github.com/ceph/ceph/blob/reef/qa/tasks/cephfs/test_nfs.py#L819-L830

Reply:
So, @phlogistonjohn mentioned in https://github.com/ceph/ceph/pull/46209#issuecomment-1400479560 that the changes weren't deliberately backported. Do you think _now_ that change should be backported to q? @dparmar18

Reply:
if the concern is only this test case then we can just backport this commit https://github.com/ceph/ceph/commit/e03331e7c4412e33fe55d7a41a22dfeba17cb463, it is nothing but adding try/except block to catch the exception

Reply:
> if the concern is only this test case then we can just backport this commit [e03331e](https://github.com/ceph/ceph/commit/e03331e7c4412e33fe55d7a41a22dfeba17cb463), it is nothing but adding try/except block to catch the exception

Yeh. I guess that's a fair exception to handle for this. Please post a backport PR @dparmar18 

Thx for looking into this.